constraint/pkg/client/clienttest/cts/opts.go

```diff
@@ -1,30 +1,33 @@
 package cts
 
 import (
+	"github.com/open-policy-agent/opa/v1/ast"
+	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions"
+
 	"github.com/open-policy-agent/frameworks/constraint/pkg/client/drivers/rego/schema"
 	"github.com/open-policy-agent/frameworks/constraint/pkg/core/templates"
 	"github.com/open-policy-agent/frameworks/constraint/pkg/handler/handlertest"
-	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions"
 )
 
 const (
 	ModuleDeny = `
 package foo
 
-violation[{"msg": msg}] {
+violation contains {"msg": msg} if {
   true
   msg := "denied"
 }
 `
-	MockTemplateName  string = "fakes"
-	MockTemplate      string = "Fakes"
-	MockTargetHandler string = "foo"
+	MockTemplateName  string          = "fakes"
+	MockTemplate      string          = "Fakes"
+	MockTargetHandler string          = "foo"
+	RegoVersion       ast.RegoVersion = ast.RegoV0
```

Comment:
Where is `RegoVersion` being used?

Reply:
I think this may have been missed. I am ok with merging this PR. Or I can raise a follow up to remove this asap. thoughts?

Reply:
hmm i think the intent might be to use it as part of the defaults https://github.com/open-policy-agent/frameworks/pull/517/files/dc39c8d9ba4422d4229d5d72fec977bb715e8d59#diff-cbc675a4ccf1d3ea615bc28ddc01ee192bcf11a9a4a206caa462cdac8454317aR30